src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,56 @@
+package controller;
+
+import domain.*;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Controller {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final LadderGenerator ladderGenerator;
+
+    public Controller(InputView inputView, OutputView outputView, LadderGenerator ladderGenerator) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.ladderGenerator = ladderGenerator;
+    }
+
+    public void run() {
+        Users usersName = settingUsers();
+        Ladders ladders = settingLadders();
+        ladders.make(usersName.getCount() - 1, ladderGenerator);
+        outputView.printUsers(usersName);
+        for (Ladder ladder : ladders.getLadders()) { //스트림
```

Comment:
```suggestion
        ladders.getLadders().forEach(outputView::printLadder);
```
요렇게 바꾸실 생각이셨던 걸까요?ㅎㅎ 개인적으로는 이 부분이 OutputView로 넘어가는건 어떨까 싶어요!

Controller.java
```
public void run() {
      Users usersName = settingUsers();
      Ladders ladders = settingLadders();
      ladders.make(usersName.getCount() - 1, ladderGenerator);
      outputView.printUsers(usersName);
      outputView.printLadders(ladders);
  }
```

OutputView.java
```
public void printLadders(Ladders ladders) {
      for (Ladder ladder : ladders.getLadders()) {
          printLadder(ladder);
      }
  }
```